packages/st2/debian/rules

```diff
@@ -13,16 +13,41 @@ DH_VIRTUALENV_INSTALL_ROOT := /opt/stackstorm
 export DH_VIRTUALENV_INSTALL_ROOT
 
 IS_SYSTEMD = $(shell command -v dh_systemd_enable > /dev/null 2>&1 && echo true)
+DEB_DISTRO := $(shell lsb_release -cs)
+
+# Trusty, Xenial and Bionic depend on different libraries.
+# Most important, Bionic uses Python 3 and others use Python 2.7
+BIONIC_PRE_DEPENDS := python3 (>= 3.6)
+BIONIC_DEPENDS := python3-distutils, python3-dev
+
+OTHER_PRE_DEPENDS := python2.7
+OTHER_DEPENDS = libpython-dev | python-dev
+
+ifeq ($(DEB_DISTRO),bionic)
+	BUILD_PRE_DEPENDS := $(BIONIC_PRE_DEPENDS)
```

Comment:
Because `BIONIC_PRE_DEPENDS` and similar vars are not used everywhere more than 1 time, can we just simplify it to something like

```suggestion
	BUILD_PRE_DEPENDS := python3 (>= 3.6)
```

This will move real values closer to context, making it easier to understand the flow.
Otherwise var->var->var dependency is too Pythonic :)